docs/branches.md

```diff
@@ -11,48 +11,45 @@ pass a branch's name to commands that want a revision as argument. For example,
 `jj branch list` to list branches and `jj branch` to create, move, or delete
 branches. There is currently no concept of an active/current/checked-out branch.
 
-## Remotes
-
-Jujutsu records the last seen position on each remote (just like Git's
-remote-tracking branches). You can refer to these with
-`<branch name>@<remote name>`, such as `jj new main@origin`.
-
-A remote branch can be associated with a local branch of the same name. It's
-sometimes called a tracking branch. When you pull from a remote, any changes
-compared to the current record of the remote's state will be propagated to the
-tracking local branch. Let's say you run `jj git fetch --remote origin` and the
-remote's "main" branch has moved so its target is now ahead of the local record
-in `main@origin`. That will update `main@origin` to the new target. It will also
-apply the change to the local branch `main`. If the local target had also moved
-compared to `main@origin` (probably because you had run `jj branch set main`),
-then the two updates will be merged. If one is ahead of the other, then that
-target will be the new target. Otherwise, the local branch will be conflicted
-(see the next "Conflicts" section for details).
-
-Most commands don't show the tracking remote branch if it has the same target as
+## Remotes and tracked branches
+
+Jujutsu records the last seen position of a branch on each remote (just like
+Git's remote-tracking branches). This record is updated on every `jj git fetch`
+and `jj git push` of the branch. You can refer to the remembered remote branch
+positions with `<branch name>@<remote name>`, such as `jj new main@origin`. `jj`
+does not provide a way to manually edit these recorded positions.
+
+A remote branch can be associated with a local branch of the same name. This is
+called a **tracked remote branch**. When you pull a tracked branch from a
+remote, any changes compared to the current record of the remote's state will be
+propagated to the corresponding local branch, which will be created if it
+doesn't exist already.
+
+!!! note "Details: how `fetch` pulls branches"
+
+    Let's say you run `jj git fetch --remote origin`. Let's also say that `jj` determines
```

Comment:
Nit: Can you keep the first two sentences logically consistent? Something like "Let's say you run `jj git fetch --remote origin`. And after the fetch `jj` determines ..."

Reply:
Yeah, Yuya was also unhappy with this phrasing. Here's another try.